examples/inference_model_handlers/instructor/README.md

```diff
@@ -0,0 +1,31 @@
+## Model Description
+
+The [instructor-large](https://huggingface.co/hkunlp/instructor-large) and [instructor-xl](https://huggingface.co/hkunlp/instructor-xl) models are both instruction-finetuned text embedding models developed by the NLP Group of The University of Hong Kong. The instructor model series is state-of-the-art on the Massive Text Embedding Benchmark (MTEB).
+
+## Deploy
+
+To deploy the instructor-large model, run: `mcli deploy -f instructor_large.yaml --cluster <your_cluster>`
+
+To deploy the instructor-xl model, run: `mcli deploy -f instructor_xl.yaml --cluster <your_cluster>`.
+
+## Input
```

Comment:
Could you briefly describe the input format here? can point to the instructor readme for more info https://github.com/HKUNLP/instructor-embedding